src/compiler/output-targets/dist-custom-elements/custom-elements-types.ts

```diff
@@ -97,6 +98,22 @@ const generateCustomElementsTypesOutput = async (
     `  rel?: (el: EventTarget, eventName: string, listener: EventListenerOrEventListenerObject, options: boolean | AddEventListenerOptions) => void;`,
     `}`,
     `export declare const setPlatformOptions: (opts: SetPlatformOptions) => void;`,
+    ...(isBundleExport
+      ? [
+          ``,
+          `/**`,
+          ` * Utility to define all custom elements within this package using the tag name provided in the component's source.`,
+          ` * When defining each custom element, it will also check it's safe to define by:`,
+          ` *`,
+          ` * 1. Ensuring the "customElements" registry is available in the global context (window).`,
+          ` * 2. The component tag name is not already defined.`,
```

Comment:
small wording suggestion:

```suggestion
          ` * When defining each custom element, it will also check it's safe to define by:`,
          ` *`,
          ` * 1. Ensuring the "customElements" registry is available in the global context (window).`,
          ` * 2. Ensuring that the component tag name is not already defined.`,
```